packages/timer/src/index.test.ts

```diff
@@ -1,113 +1,111 @@
-import { test } from 'uvu'
-import * as assert from 'uvu/assert'
+import { describe, test, expect } from 'vitest'
 import { createTestCtx, getDuration } from '@reatom/testing'
 import { sleep } from '@reatom/utils'
-
 import { reatomTimer } from './'
 
-test(`base API`, async () => {
-  const timerAtom = reatomTimer(`test`)
-  const ctx = createTestCtx()
-
-  timerAtom.intervalAtom.setSeconds(ctx, 0.001)
-
-  var target = 50
-  var duration = await getDuration(() =>
-    timerAtom.startTimer(ctx, target / 1000),
-  )
+describe(
```

Comment:
It would be super nice (not required) to rewrite these tests with a fake timers, instead of magic sleep numbers and it derivatations.

Reply:
I tried but no luck :(
let's try in new PR later on